app/scheduler/modules/pairing.py

```diff
@@ -0,0 +1,63 @@
+"""Match two Launchpad member for a private conversation"""
+from slack import WebClient
+from interface.slack import Bot
+from random import shuffle
+from .base import ModuleBase
+from typing import Dict, List, Tuple, Any
+from flask import Flask
+from config import Config
+from db.facade import DBFacade
+import logging
+
+
+class Pairing(ModuleBase):
+    """Module that matches 2 launchpad members each week"""
+
+    NAME = 'Match launch pad members randomly'
+
+    def __init__(self,
+                 flask_app: Flask,
+                 config: Config,
+                 facade: DBFacade):
+        """Initialize the object."""
+        self.bot = Bot(WebClient(config.slack_api_token),
+                       config.slack_notification_channel)
+        self.channel_id = self.bot.get_channel_id(config.slack_pairing_channel)
+        self.facade = facade
+
+
+    def get_job_args(self) -> Dict[str, Any]:
+        """Get job configuration arguments for apscheduler."""
+        return {'trigger':      'cron',
+                'minute': '*',
```

Comment:
I'm guessing this is just temporary for testing, but could also just make this a configuration option! `SLACK_PAIRING_FREQUENCY` for example

Reply:
Alrighty made it an env variable as a cron job string... not the cleanest so happy for recommendations!